src/components/Navigation.astro

```diff
@@ -39,7 +40,7 @@ const featuredSponsors = DATA.sponsors.filter(
     max-xl:duration-0"
     tabindex="-1"
   >
-  <ul
+    <ul
```

Comment:
max-xl:gap-8 is dupped

Reply:
[Fixed](https://github.com/biznaga-fest/web/pull/10/commits/b41626f066c5ff9591d57004ec19dea1a202170a)